### DISCUSSION TITLE
Welcome to Travel-Ui Discussions!

### DISCUSSION THREAD

**Discussion by moha-b** [moha-b/Travel-Ui#2] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by player06** [comment, marked as answer]
how about some fun in this shitt

**Comment by moha-b** [comment]
no
